### ISSUE TITLE
More minigames

### ISSUE THREAD

**Issue by bobbens** [naev/naev#1945]
For the Za'lek campaign, having lots of puzzle minigames seems like a good way to go. Right now a Sokoban minigame is implemented, but it would be nice to have more variety and thematically fitting games. As @UncombedCoconut pointed out, https://www.chiark.greenend.org.uk/~sgtatham/puzzles/ might be a good source of ideas. In particular, [net](https://www.chiark.greenend.org.uk/~sgtatham/puzzles/js/net.html) and [signpost](https://www.chiark.greenend.org.uk/~sgtatham/puzzles/js/signpost.html) might be fairly good candidates. License is MIT.

**Comment by seularts** [comment]
This is a cool sliding puzzle: https://appadvice.com/app/unblock-slide-puzzle-free-slide-block-bloxx-puzzle/1142553199
But you need to use the mouse to drag the pieces.

Also, for the casino, Royal Game of Ur would be a cool addition as it is an easy game to learn but hard to master!
https://royalur.net/

It's also super addictive and fun! :D

**Comment by bobbens** [comment]
Yeah, I think we want to still stay away from mouse stuff for now. Hopefully allowing for proper full joystick control. Royal Game of Ur looks like it could work somewhere potentially, but it doesn't really feel like a betting game for the casino.

**Comment by bobbens** [comment]
Follow-up, new mining rhythm mini-game was added for mining. Still need more variety and types for other things though (mainly missions).

**Comment by bobbens** [comment]
Two new mini-games were added. One is used for Points of Interest, the other still has to be used.

**Comment by bobbens** [comment]
I'm guessing we have enough to close this issue for now. Thanks!
